mmv1/templates/terraform/examples/data_fusion_instance_full.tf.erb

```diff
@@ -18,7 +18,10 @@ resource "google_data_fusion_instance" "<%= ctx[:primary_resource_id] %>" {
     network       = "default"
     ip_allocation = "${google_compute_global_address.private_ip_alloc.address}/${google_compute_global_address.private_ip_alloc.prefix_length}"
   }
-
+  accelerators {
+    accelerator_type = "CDC"
+    state = "DISABLED"
```

Comment:
Do we want to change this to "ENABLED". As this example will also be generated in our doc, it may make more sense if the config is used to enable the feature in the example usage. But it's totally your call!

Reply:
Since I needed to go ahead and do the other update I made this one too!